packages/local_auth/local_auth_darwin/lib/src/messages.g.dart

```diff
@@ -36,15 +36,16 @@ enum AuthResult {
   errorPasscodeNotSet,
 }
 
-/// Pigeon equivalent of the subset of BiometricType used by iOS.
+/// Pigeon equivalent of the subset of BiometricType used by iOS & MacOS.
```

Comment:
"and"

Same throughout this file (and the rest of the PR if there are other instances).